.travis.yml

```diff
@@ -3,17 +3,26 @@ language: node_js
 os: linux
 
 matrix:
-  include:   
-    - node_js: '8'
-    - node_js: '10'
+  include:
+  - node_js: '8'
+  - node_js: '10'
 
 sudo: false
 
 install:
-  - npm install
-
+- npm install
 script:
-  - npm test
+- npm test
 
 after_success:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
+- cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
```

Comment:
Does this push the coverage report up to coveralls automatically?

Reply:
Yes, it seems automagic, but our coverage badge is still `unknown`, and I'm guessing that's because we haven't had it run on `master` yet.

Reply:
@mydiemho might know, but my guess is that it targets a default branch (master). I wonder if that's something we can change? Everything will get there eventually...